File: Programming-languages-and-laboratory/Python/Exercises/22_compito_java_20180622/Laboratorio.py
from Corso import Corso


class Laboratorio(Corso):
    def __init__(self, codice, tipo, nome, nome_cognome_docente, nome_laboratorio, nome_assistente, numero_postazioni):
        super().__init__(codice, tipo, nome, nome_cognome_docente)
        self._nome_laboratorio = nome_laboratorio
        self._nome_assistente = nome_assistente
        self._numero_postazioni = numero_postazioni

    def __str__(self):
        return super().__str__() + "\t-\t-\t-\t" + self._nome_laboratorio + "\t" + self._nome_assistente + "\t" + str(self._numero_postazioni)

File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Java/Exercises/22_compito_java_2018_06_22/Laboratorio.java
public class Laboratorio extends Corso {
	private String nomeLaboratorio, nomeAssistente;
	private int numeroPostazioniLaboratorio;
	
	public Laboratorio(int codice, String tipo, String nomeCorso, String nomeCognomeDocente, String nomeLaboratorio, String nomeAssistente, int numeroPostazioniLaboratorio) {
		super(codice, tipo, nomeCorso, nomeCognomeDocente);
		this.nomeLaboratorio = nomeLaboratorio;
		this.nomeAssistente = nomeAssistente;
		this.numeroPostazioniLaboratorio = numeroPostazioniLaboratorio;
	}
	
	public String toString() {
		return nomeCorso + "\t" + codiceCorso + "\t" + nomeCognomeDocente + "\t" + tipo + "\t-\t-\t-\t" + nomeLaboratorio + "\t" + nomeAssistente + "\t" + numeroPostazioniLaboratorio; 
	}
}